modules/core/shared/src/main/scala/EntryPoint.scala

```diff
@@ -14,20 +14,32 @@ import cats.effect.Resource
 trait EntryPoint[F[_]] {
 
   /** Resource that creates a new root span in a new trace. */
-  def root(name: String): Resource[F, Span[F]]
+  final def root(name: String): Resource[F, Span[F]] = root(name, Span.Options.Defaults)
+
+  def root(name: String, options: Span.Options): Resource[F, Span[F]]
```

Comment:
Would be good to add some docs about the use cases for `options` here. Nothing too extensive.